WebRoot/js/ajax/dwt/widgets/DwtListView.js

```diff
@@ -47,6 +47,13 @@ DwtListView = function(params) {
 	params.className = params.className || "DwtListView";
 	DwtComposite.call(this, params);
 
+	if (this.tableView && params.headerList) {
+		this.column = [];
+		for (var i = 0; i < params.headerList.length; i++) {
```

Comment:
`for (var i = 0, length = params.headerList.length; i < length; i++) {`

Reply:
Done.